### PR TITLE
Create story completed

### PR DESCRIPTION
**Pull request by burra162**

### CODE REVIEW

File: src/views/AdminPage.vue [hunk 1]
```diff
@@ -177,13 +187,22 @@ function getStories() {
...
+            <v-card-title class="headline">{{ story.title }}
+              <v-chip class="ma-2" color="primary" label>
+                {{ story.genre }}
+              </v-chip>
+            </v-card-title>
+            <v-card-text v-if="story.description.length > 0" class="single-line-text">
+              {{ story.description }}
+            </v-card-text>
+            <v-card-text v-else class="single-line-text">
+              Start using the AI to generate a story.
```
Comment (n-grounds) [inline src/views/AdminPage.vue:200]: seems like a weird alternative text for a story without a description, why would a story not have a description in the first place?

File: src/views/AdminPage.vue [hunk 2]
```diff
@@ -157,7 +157,17 @@ function getStories() {
 }
 
 
+const previewStory = ref({});
+const previewDialog = ref(false);
```
Comment (n-grounds) [inline src/views/AdminPage.vue:161]: other than lines 164 and 169 where the boolean is toggled, I don't see where `previewDialog` is ever used

File: src/views/AdminPage.vue (hunk 1 above)
Comment (burra162) [reply src/views/AdminPage.vue:200]: The plan is that we create a story with a title and the use Cohere to add the actual story into it using Cohere.
That way each story will have a separate chat history associated with it, so we can edit that in future if needed.

File: src/views/AdminPage.vue (hunk 2 above)
Comment (burra162) [reply src/views/AdminPage.vue:161]: I added this as a base to show the preview dialog. 
The remaining work is continued in next stories.

**Pull request closed without merging**